scripts/india_nfhs/README.md

```diff
@@ -0,0 +1,13 @@
+# National Family Health Survey - India
+
+## About
+The National Family Health Survey (NFHS) is a large-scale, multi-round survey conducted in a representative sample of households throughout India. National Family Health Survey (NFH) provides information on fertility, infant and child mortality, the practice of family planning, maternal and child health, reproductive health, nutrition, anaemia, utilization and quality of health and family planning services in a state as well as pan India level. The specific goal of the National Family Health Survey (NFHS) are a) to provide essential data on health and family welfare for policy and programme purposes, and b) to provide information on important emerging health and family welfare issues. Besides providing evidence for the effectiveness of ongoing programs, the National Family Health Survey (NFHS-5) data helps identify the need for new programmes with an area-specific focus and identifying groups that are most in need of essential services.
+
+## NDAP Data Portal
+There are 3 datasets available for National Family Health Survey:-
+1) [National Family Health Survey - 4 & 5: State](https://ndap.niti.gov.in/dataset/6821)
+2) [National Family Health Survey - 4 : District](https://ndap.niti.gov.in/dataset/7034)
+3) [National Family Health Survey - 5 : District](https://ndap.niti.gov.in/dataset/6822)
+
+## Dataset in this directory
+This import directory has National Family Health Survey data for all Indian states and districts for year 2015-2016 and 2019-2020. The data can be found in [data/](./data/) directory.
```

Comment:
A few notes:

- Please use [Git LFS](https://git-lfs.github.com/) for large files
- I see you've included 2 copies of each raw data file (one in the data directory and one in the states/districts directories): let's remove one of the copies and put the raw data in a folder called source_data